src/Microsoft.IdentityModel.Tokens.Saml/Saml2/Saml2Assertion.cs

```diff
@@ -84,6 +89,46 @@ public Saml2Conditions Conditions
             set;
         }
 
+        /// <summary>
+        /// Gets the canonicalized (ExclusiveC14n) representation without comments.
```

Comment:
Perhaps we should document that null will be returned in a case that we can't get the canonical string (we swallow all exceptions).

Reply:
See above comment with SamlAssertion.